solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/TestFeatureStore.java

```diff
@@ -41,6 +40,12 @@ public static void setup() throws Exception {
   }
 
   @Test
```

Comment:
Since we depend on getFeatureStore(null) returning the default feature store, can we have a unit test verifying that too?